test/asyncMotionProfileControllerTests.cpp

```diff
@@ -21,12 +21,31 @@ class MockAsyncMotionProfileController : public AsyncMotionProfileController {
     AsyncMotionProfileController::executeSinglePath(path, std::move(rate));
   }
 
+  static std::string makeFilePath(std::string directory, std::string filename) {
```

Comment:
What is the purpose of these redefinitions? If you just want to make them public you can `using` them like above.